bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/WemoBindingConstants.java

```diff
@@ -109,6 +109,7 @@
 
     // List of thing configuration properties
     public static final String UDN = "udn";
+    public static final String IPADDRESS = "ipaddress";
```

Comment:
I would suggest to camel-case the string to "ip**A**ddress" for readability and consistency.